### ISSUE TITLE
gopls and rls completion

### ISSUE THREAD

**Issue by mastertinner** [dense-analysis/ale#2987]
It would be great to have completion for `gopls` and `rls` (and possibly other language servers like `terraform-lsp`) like we have it for `tsserver`. Is that already possible or would we need to add that functionality? For me, it's not working out of the box and I'm not sure, if possible, how to enable it.

**Comment by RyanSquared** [comment]
It should work out of the box, I thought? Are you sure that `gopls` and `rls` are enabled? What's the output of `:ALEInfo`? Additionally, are you using Deoplete and have ALE hooked up as a source for that? See the completion section of the README (here)[https://github.com/dense-analysis/ale#usage-completion] for more information about how to do that.

**Comment by mastertinner** [comment]
Thanks for your quick reply, @RyanSquared! I'm not using Deoplete but have ALE completion enabled with `let g:ale_completion_enabled = 1`. I was actually looking for the option to enable these as completion engines but couldn't find it. However, after your comment, I enabled them as linters and it works. Sorry, I wasn't aware that I had to enable them as linters for them to work as completion engines. :) Thanks for your help!
